slate-video-library/src/Components/Context/SingleVideopageContext.js

```diff
@@ -47,18 +70,68 @@ function SingleVideopageContext({ children }) {
     getSingleVideos();
   }, [videoId]);
 
-  console.log(videoId);
-  console.log(`video data`, videoData);
+  // set likes for individual single video
+
+  const likedVideo = async (videoData, dispatch) => {
+    const response = await axios({
+      method: "POST",
+      url: `/api/user/likes`,
+      headers: { authorization: localStorage.getItem("token") },
+      data: { video: videoData },
+    });
+
+    dispatch({ type: "SINGLEVIDEOLIKES", payload: response.data.likes });
```

Comment:
use underscore or hyphens to join two words for readability like `SINGLE_VIDEO_LIKES`